libs/hpc-dummy/src/lib/dummy.ts

```diff
@@ -50,30 +50,24 @@ function dummyEndpoint<Args extends [unknown, ...unknown[]], Data>(
   fn: (...data: Args) => Promise<Data>
 ): (...args: Args) => Promise<Data> {
   return (...args: Args) =>
-    new Promise<Data>((resolve, reject) => {
+    new Promise<Data>((resolve) => {
       console.log('[DUMMY] Endpoint Called: ', name, ...args);
-      if (Math.random() > 0.1) {
-        setTimeout(
-          () =>
-            resolve(
-              fn(...args).then((data) => {
-                console.log(
-                  '[DUMMY] Endpoint Resolving: ',
-                  name,
-                  ...args,
-                  data
-                );
-                return data;
-              })
-            ),
-          300
-        );
-      } else {
-        setTimeout(() => reject(new Error('A random error ocurred!')), 300);
-      }
+      // TODO: allow triggering dummy endpoint failures for testing
+      setTimeout(
+        () =>
+          resolve(
+            fn(...args).then((data) => {
+              console.log('[DUMMY] Endpoint Resolving: ', name, ...args, data);
+              return data;
+            })
+          ),
+        300
+      );
     });
 }
 
+export type { DummyData };
```

Comment:
We re-export this type. Can't we import it elsewhere the same way we do in this file?

Reply:
Done